Non-Interactive-Protocol.md

```diff
@@ -0,0 +1,173 @@
+# Non-interactive Onchain Protocol
+
+## Abstract
+
+DLC allows two parties to conduct stateful bitcoin contracts by giving each
+of the parties *cross-signed range of CET transactions*. These CET transactions
+are all spending the same output from a funding transaction and thus only one of
+the alternative can confirm. A CET is finalized according to the outcome signed
+by the previously selected oracle.
+
+As the contract is established once funding transaction signatures have been exchanged,
+from then a party should assume counterparty non-cooperation and ensure either
+confirmation of the funding or double-spend of its collatral inputs to avoid exploitation.
+
+In the optimistic way, once the oracle signature is released, parties interact
+off-chain to settle the DLC. They generate a *closing_tranaction* and publish it on the
+blockchain.
+
+In the pessimitic way, a party interact with the chain to unilaterally settle the DLC.
+They are two scenarios to consider:
+
+1.  The oracle releases the outcome signature but the DLC counterparty goes wrong,
+either going offline or maliciously behaving. The party must broadcast the DLC
+transaction paired with the outcome.
```

Comment:
With adaptor version, that's not the case anymore. There is no need for interaction and there is no closing transaction. They just use the oracle signature to decrypt one of the adaptor signature from their counter party and any of them can broadcast the corresponding CET.

So AFAIK as soon as the fund transaction is confirmed, there is no way for any party to misbehave (apart from colluding with the oracle but that's another story).

Reply:
Well, they can attempt to delay the CET and broadcast the refund transaction.

Reply:
Right we don't have a closing transaction anymore. Deleted. 

Note, a party can misbehave by a bunch of way before or after oracle signature release like time-dilating the honest party or DoSing its node until refund timeout.